featurebyte/query_graph/node/base.py

```diff
@@ -972,34 +1057,95 @@ def _derive_sdk_code(
             node_inputs, var_name_generator, ClassEnum.PD_TIMESTAMP, self.generate_expression
         )
 
+    def _generate_odfv_expression_with_null_value_handling_for_single_input(
+        self, left_operand: str, right_operand: str
+    ) -> str:
+        expr = self.generate_odfv_expression(left_operand, right_operand)
+        where_expr = f"np.where(pd.isna({left_operand}), np.nan, {expr})"
+        return f"pd.Series({where_expr}, index={left_operand}.index)"
+
+    def _generate_odfv_expression_with_null_value_handling(
+        self, left_operand: str, right_operand: str
+    ) -> str:
+        expr = self.generate_odfv_expression(left_operand, right_operand)
+        where_expr = f"np.where(pd.isna({left_operand}) | pd.isna({right_operand}), np.nan, {expr})"
+        return f"pd.Series({where_expr}, index={left_operand}.index)"
```

Comment:
These two methods fix existing issue when any of the operand contains null values.